lib/hanami/helpers/form_helper/form_builder.rb

```diff
@@ -1348,8 +1348,25 @@ def datalist(name, values, list, attributes = {}) # rubocop:disable Metrics/Meth
         #
         #   <!-- output -->
         #   <button class="btn btn-secondary">Click me</button>
-        def button(content, attributes = {})
+        #
+        # @example block
+        #   <%=
+        #     # ...
+        #     button class: "btn btn-secondary" do
+        #       span class: 'oi oi-check'
+        #     end
+        #   %>
+        #   <!-- output -->
+        #   <button class="btn btn-secondary">
+        #     <span class="oi oi-check"></span>
+        #   </button>
+        def button(content, attributes = {}, &blk)
           # This is here only for documentation purposes
```

Comment:
I think you ought to remove this line, since it's not true any more... ;)

Reply:
it's ready